app/pages/stats/page.tsx

```diff
@@ -5,10 +5,18 @@ import TabsComponent from "./components/tabsComponent";
 import StatCard from "./components/statCard";
 import ActivityBreakdown from "./components/activityBreakdown";
 import chroma from "chroma-js";
+import { useUserId } from "@/hooks/user/useUserId";
+import { useProfile } from "@/hooks/user/useProfile";
+import { isSameUser } from "@/utils/authorization";
 
 const StatsPage: React.FC = () => {
   const { userId } = useParams();
-  const { stats, isLoading } = useStats(Number(userId));
+  const { stats, isLoading, isForbidden } = useStats(Number(userId));
+  const loggedUserId = useUserId();
```

Comment:
i dont get it , useUserId returnes  loggedUserId ?  probably useUserId should be renamed